File: src/main/java/nu/peg/slack/pt/api/transport/model/Journey.java
package nu.peg.slack.pt.api.transport.model;

import java.util.List;

import lombok.Data;

@Data
public class Journey {

    private String name, category, number, operator, to;
    private Integer categoryCode;
    private List<Checkpoint> passList;
    private Integer capacity1st, capacity2nd;

}

File: src/main/kotlin/nu/peg/slack/pt/api/transport/model/Journey.kt
package nu.peg.slack.pt.api.transport.model

import lombok.Data

@Data
class Journey {

    private val name: String? = null
    private val category: String? = null
    private val number: String? = null
    private val operator: String? = null
    private val to: String? = null
    private val categoryCode: Int? = null
    private val passList: List<Checkpoint>? = null
    private val capacity1st: Int? = null
    private val capacity2nd: Int? = null

}
